src/pages/Waiting/index.js

```diff
@@ -1,77 +1,86 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 
 export default function WaitingRoom() {
+  const [isSelected, setIsSelected] = useState(false);
+  const [isReady, setIsReady] = useState(false);
+  const toggleIsSelected = () => setIsSelected(!isSelected);
+  const toggleIsReady = () => setIsReady(!isReady);
```

Comment:
지금처럼 간단한 것은 한 줄로도 가능하나 . 
아래처럼 2줄로 중괄호로 표현 하는게 명시적으로 좋을 것 같다는 리뷰를 받았었습니다. 이거는 개인 취향이라 참조만 해주시면 될것 같습니다.
```suggestion
  const toggleIsSelected = () => {
    setIsSelected(!isSelected);
  }
  
  const toggleIsReady = () => {
    setIsReady(!isReady);
    }
```